src/YUILoader.cc

```diff
@@ -156,6 +167,48 @@ void YUILoader::loadUI( bool withThreads )
     }
 }
 
+void YUILoader::loadRestAPIPlugin( const std::string & wantedGUI, bool withThreads )
+{
+    // Do not try to load if variable is not set
+    yuiMilestone () << "Requested to start http server to control UI." << std::endl;
+    if( pluginExists( YUIPlugin_RestAPI ) )
+    {
+        // Load underlying UI plugin, as test method inherits from it
+        // YUIPlugin_Test uses both libraries to be single point of entry
```

Comment:
I'd add a `TODO` or a `FIXME` comment here, loading both UIs is acceptable in the initial implementation, but in the long term we should avoid it.